sbin/get-redisjson

```diff
@@ -149,16 +151,20 @@ WORK_DIR=$(mktemp -d /tmp/${MOD_NAME}.XXXXXX)
 if [[ -z $REPO_PATH ]]; then
 	F_MOD_RAMP="${WORK_DIR}/${MOD_RAMP}"
 	if [[ $FORCE == 1 || ! -f $F_MOD_RAMP ]]; then
-		echo "Download ${MOD_NAME} [${MOD_S3_URL}/${MOD_RAMP}] ..."
-		runn wget -P ${WORK_DIR} ${MOD_S3_URL}/${MOD_RAMP}
-		if [[ $NOP != 1 && $? != 0 ]]; then
-			if [[ BUILD == 0 ]]; then
-				eprint "Cannot download binary artifacts - build ${MOD_NAME} and set REPO_PATH"
-				exit 1
+		# check if redisjson module exists in the s3 bucket
+		if wget -q --spider ${WORK_DIR} ${MOD_S3_URL}/${MOD_RAMP}; then
```

Comment:
so it's just a prettier validation, not a solution in case it is missing right?

Reply:
The condition was wrong, but now it is fixed.
If the module exists, it is downloaded, if the module is not found, it is build from source code.